### PR TITLE
Create sign up page with style

### LINKED ISSUE #120: [React] Sign Up form

### PR DESCRIPTION
**Pull request by tariqabusada**

### CODE REVIEW

File: client/src/components/signUpForm/index.tsx [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+          variant="contained"
+          fullWidth
+          type="submit"
+        >
+          Sign Up
+        </Button>
+        <h4 className="another-way">
+          Already you have an account!
+          {' '}
+          <a href="/login">Log In</a>
```
Comment (husamkamal) [inline client/src/components/signUpForm/index.tsx]: use a link from react-router 
```suggestion
          <a href="/login">Log In</a>
          <Link to='/login'>Login In </Linl>
```

**Review by husamkamal — CHANGES_REQUESTED** [review]
Screenshot from 2022-10-30 08-14-10

add space between field

File: client/src/pages/Login/index.tsx [hunk 2]
```diff
@@ -34,13 +34,13 @@ function Login() {
   });
 
   return (
-    <div className="loginPage">
+    <div className="signPage">
```
Comment (LinaYahya) [inline client/src/pages/Login/index.tsx]: not sure why you change classNames

File: client/src/pages/Login/style.css [hunk 3]
```diff
@@ -15,11 +15,11 @@
   font-size: 30px;
   font-weight: bold;
 }
-.loginPage{
+.signPage{
```
Comment (LinaYahya) [inline client/src/pages/Login/style.css]: again not sure why you change `className`'s

File: client/src/components/signUpForm/index.tsx [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+
+  return (
+    <div className="formPage">
+      <form className="signup-form" onSubmit={forma.handleSubmit}>
+        <h1>Sign Up</h1>
+        <TextField
+          fullWidth
+          id="fullName"
+          name="fullName"
+          label="fullName"
```
Comment (LinaYahya) [inline client/src/components/signUpForm/index.tsx]:
```suggestion
          label="Full Name"
```

File: client/src/components/signUpForm/index.tsx [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+          value={forma.values.email}
+          onChange={forma.handleChange}
+          error={forma.touched.email && Boolean(forma.errors.email)}
+          helperText={forma.touched.email && forma.errors.email}
+        />
+        <TextField
+          fullWidth
+          id="phoneNumber"
+          name="phoneNumber"
+          label="phoneNumber"
```
Comment (LinaYahya) [inline client/src/components/signUpForm/index.tsx]:
```suggestion
          label="Phone Number"
```

File: client/src/components/signUpForm/index.tsx [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+          value={forma.values.phoneNumber}
+          onChange={forma.handleChange}
+          error={forma.touched.phoneNumber && Boolean(forma.errors.phoneNumber)}
+          helperText={forma.touched.phoneNumber && forma.errors.phoneNumber}
+        />
+        <TextField
```
Comment (LinaYahya) [inline client/src/components/signUpForm/index.tsx:85]: create a field to confirm password, just to check if passwords confirmed
and no need to send it to the server

File: client/src/components/signUpForm/index.tsx [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+          name="password"
+          label="Password"
+          type="password"
+          value={forma.values.password}
+          onChange={forma.handleChange}
+          error={forma.touched.password && Boolean(forma.errors.password)}
+          helperText={forma.touched.password && forma.errors.password}
+        />
+        {
+          resError
```
Comment (LinaYahya) [inline client/src/components/signUpForm/index.tsx:108]: try to clear error, when email chnaged

File: client/src/pages/SignUp/index.tsx [hunk 8]
```diff
@@ -1,5 +1,22 @@
+import SignUpform from '../../components/signUpForm';
+import '../Login/style.css';
```
Comment (LinaYahya) [inline client/src/pages/SignUp/index.tsx:2]:
```suggestion
```
No need to import style as it's imported in another place

**Review by LinaYahya — CHANGES_REQUESTED** [review]

File: client/src/pages/Login/index.tsx (hunk 2 above)
Comment (tariqabusada) [reply client/src/pages/Login/index.tsx]: to use one style file in two pages: login and signup

**Review by husamkamal — APPROVED** [review]

**Review by abdou059 — APPROVED** [review]

**Review by mai-mohammed — APPROVED** [review]

**Pull request merged**